index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```

Comment:
- [ ] As suggested by the previous reviewer, you'll need to add the class used in your css file to your html markup to make sure that your style.css works.

## EXAMPLE
`<h1 class="header">Hello Microverse</h1>`